packages/styles/scss/patterns/listsection/_listsection-group.scss

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection-group {
+    .#{$prefix}--listsection-group {
+    color: $text-01;
+    &__title {
+      @include type-style('expressive-heading-04');
+    }
+    @include carbon--breakpoint('sm') {
+      &__col {
+        padding-left: $layout-01;
+        padding-right: $layout-01;
+      }
```

Comment:
This shouldn't be necessary from what I'm seeing in the spec. Just have one containing column, and let that handle things.